nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -2271,6 +2283,7 @@ class GreedyRNNTInferConfig:
     preserve_alignments: bool = False
     preserve_frame_confidence: bool = False
     confidence_method_cfg: Optional[ConfidenceMethodConfig] = field(default_factory=lambda: ConfidenceMethodConfig())
+    use_cuda_graph_decoder: bool = False
```

Comment:
Please, add this parameter to `GreedyBatchedRNNTInferConfg`

Reply:
You're right. I was super confused between GreedyRNNTInferConfig and GreedyBatchedRNNTInferConfig. I can't even find any uses of GreedyRNNTInferConfig in the code base. So it should probably be removed.